src/components/Preimages/ParamsInput.tsx

```
@@ -0,0 +1,479 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React from 'react';
+import { Form, Input, Tooltip, Switch, Select, Button, InputNumber } from 'antd';
+import { InfoCircleOutlined, PlusOutlined, MinusCircleOutlined } from '@ant-design/icons';
+import { IPreimageArgument } from '~src/types';
+import styled from 'styled-components';
+import { parseParamType, ParamType, ITupleType } from './types';
```

ESLint

[error] 'ITupleType' is defined but never used.

(@typescript-eslint/no-unused-vars)